setup.py

```diff
@@ -73,6 +73,7 @@
         'dev': dev_extras,
         'docs': docs_extras,
         'tests': testing_requires,
+        'mypy': ['mypy']
```

Comment:
Presumably this should be under `dev_extras` so it's included in the `deps = -e .[dev]` call; is this called somewhere else that I'm not seeing?